### ISSUE TITLE
Assertion at..., condition '<disabled>' not met.

### ISSUE THREAD

**Issue by CincySteve** [dotnet/aspnetcore#25957]
I just published to Azure a Blazor Wasm SPA I am developing with VS Community 2019.  The app starts up successfully in Chrome via the VS debugger, but the published version triggers the exception below (also in Chrome).  The exception message, _ gives me no clue as to the problem_.  This looks similar to this [previous issue #16818](https://github.com/mono/mono/issues/16818) that has been fixed.  I published 2-3 months ago a much simpler early version of the app, which functionally did what's described below, and it worked at that time.  

Curiously, after I first published a few minutes ago, my console log messages indicated the app got through almost all of my main layout's OnInitializedAsync code, including a JSRuntime.InvokeAsync to my JavaScipt initialization code that opens an IndexedDB and returns its Index of keys.  But when I tried re-publishing and got the same exception, none of my initialization code ran according to the console log.  I can post my OnInitializedAsync code, but since the exception is now happening before any of this code executes, I'm not sure it will be helpful.  Any idea what's going on?

Thanks.  Steve

![App Startup Exception](https://user-images.githubusercontent.com/18602722/93339179-441ead80-f7f9-11ea-9720-a27c236dfd58.JPG)

**Comment by mkArtakMSFT** [comment]
Thanks for contacting us.
Can you try out [.NET 5.0 RC1](https://dotnet.microsoft.com/download/dotnet/5.0) SDK and see whether you will meet the same error or not?

**Comment by dantronik-markus** [comment]
I am getting the same error with .NET 5.0 RC1. Here's the error message:
![2020-09-17 10_16_30-Entwicklerwerkzeuge - IKON_2_0 - http___192 168 1 53_84_](https://user-images.githubusercontent.com/67917640/93439725-30457b00-f8cf-11ea-89a6-211970ec2403.png)

**Comment by modulardev2** [comment]
Yep, same issue here. Only in the published version, everything works fine locally.

**Comment by lertoo** [comment]
Same issue here with RC1

**Comment by modulardev2** [comment]
After messing with this for awhile today, I was able to fix it on my end using the following workaround, and I hope it can help someone else.

The culprit was the HttpClient.GetStringAsync method. I replaced for example:

`await HttpClient.GetStringAsync(page)`

with

`await (await HttpClient.GetAsync(page)).Content.ReadAsStringAsync();`

**Comment by lertoo** [comment]
> After messing with this for awhile today, I was able to fix it on my end using the following workaround, and I hope it can help someone else.
> 
> The culprit was the HttpClient.GetStringAsync method. I replaced for example:
> 
> `await HttpClient.GetStringAsync(page)`
> 
> with
> 
> `await (await HttpClient.GetAsync(page)).Content.ReadAsStringAsync();`

It works. You saved my day. Thanks.

**Comment by dantronik-markus** [comment]
@mkArtakMSFT Can auto-close on this issue be prevented? It's a valid issue and the original reporter not responding does not mean that no one else of us will.

**Comment by CincySteve** [comment]
Sorry for being so slow to get back in.  I installed .Net 5.0 RC1.  I verified that I am using VS 16.8, but am not sure if there is some other verification required to confirm other 5.0 RC1 things installed.  Anyway, assuming I have done what was necessary, the app still fails to start-up because of the same exception.

**Comment by CincySteve** [comment]
Ugh.  I just started to run into this exception in the VS debugger, in both v16.7.4 and v16.8 (.Net 5.0 RC1).  That means I am dead in the water until I can either find the cause or a workaround.  I could sure use some help.

**Comment by CincySteve** [comment]
I'm guessing it's no surprise, but when I remove the statement below from index.html, startup hangs, but the exception does not occur.  So the problem would seem to be in this JS.

<script src="_framework/blazor.webassembly.js"></script>``

**Comment by CincySteve** [comment]
I have a couple other small Blazor apps which are not running into this exception.  So there is something about my one app that's triggering it.

I have tried, without success:
1. Shutting down VS (both the v16.7 and v16.8 versions) and restarting.
2. Rebooting my computer.
3. Removing all the markup from my Layout that's invoked by App.razor
4. Removing all the startup logic in my javascript.
5. Removing all the code in my Layout's OnInitializedAsync and OnAfterRender overrides.
6. Doing 3-5 together (i.e. the app would do nothing)
7. Removing both the <Found> and <NotFound> component references in App.razor

**Comment by mkArtakMSFT** [comment]
Thanks for sharing details, everyone.
Can somebody please share a minimalistic repro project so we can root cause and address this? Thanks!
